examples/Viewer.tsx

```diff
@@ -127,7 +127,7 @@ const initialState = {
     currentFrame: 0,
     currentTime: 0,
     height: 700,
-    width: 800,
+    width: 1200,
```

Comment:
widening the example viewer allows me to use the ctrl-alt-1 advanced settings without hiding so much of the view.